SMSegmentViewController/SMSegmentView/SMSegment.swift

```diff
@@ -16,10 +16,8 @@ open class SMSegment: UIView {
     // Title
     public var title: String? {
         didSet {
-            DispatchQueue.main.async(execute: {
-                self.label.text = self.title
-                self.layoutSubviews()
-            })
+            self.label.text = title
+            setNeedsLayout()
```

Comment:
I'd prefer to keep using `self.` to refer to a property/function call, despite this is sort of against "commonly accepted" Swift coding style.